acceptance/tests/ticket_1238_hostname_fqdn.rb

```diff
@@ -14,6 +14,7 @@
 
   step 'set hostname as fqdn' do
     on(agent, "hostname #{fqdn}")
+    sleep 1 # on Solaris 11 hostname returns before the hostname is updated
```

Comment:
It has been when tested locally on the Solaris 11 SPARC box.
This is a recent failure, so I don't know what has changed from a performance standpoint on the box. I am trying not to introduce any additional unneeded latency. Having said that, I am open to bumping it up.